discover/deployment/articles-dxp/01-installing-liferay/10-trial-plugin-installation.markdown

```diff
@@ -35,11 +35,11 @@ options to update or install certain plugins. To install a plugin, click the
 
 ![Click the *Install* button to install a plugin.](../../images-dxp/dxp-store-install.png)
 
-**5)** Trial users have access to the Customer Portal during their trial, with
-the exception of our customer Knowledge Base and LESA, our support ticketing
-system. This also includes access to the @product@-specific plugins from the Liferay
+**5)** Trial users have access to the Customer Portal during their trial. This
+also includes access to the @product@-specific plugins from the Liferay
 [Marketplace website](https://www.liferay.com/marketplace) on Liferay.COM
-(LRDC).
+(LRDC). Trial users will not have access to our customer Knowledge Base and
```

Comment:
Just a quick note: try to keep the future tense out of the docs; it's unnecessarily wordy.